forge/src/main/java/org/openjdk/skara/forge/github/GitHubPullRequest.java

```diff
@@ -795,4 +796,20 @@ public String limitBodySize(String body) {
         }
         return body;
     }
+
+    private ZonedDateTime lastMarkedAsReadyTime() {
+        ZonedDateTime readyTime = createdAt();
+        Optional<ZonedDateTime> latestTime = request.get("issues/" + json.get("number").toString() + "/timeline")
```

Comment:
This fetches the timeline a second time. Can you try to reuse the timeline result from lastForcePushTime()?

Could you also update the javadoc on `lastForcePushTime` that it's returning the last time something was force pushed while not in draft state.

Reply:
Good idea! I will try to reuse it. I will update the java doc too.